src/wmtk/PointMesh.cpp

```diff
@@ -49,16 +49,10 @@ void PointMesh::initialize(long count)
 }
 
 
-bool PointMesh::is_valid(const Tuple& tuple) const
+bool PointMesh::is_valid(const Tuple& tuple, ConstAccessor<long>& hash_accessor) const
 {
-    return true;
-}
-
-bool PointMesh::is_outdated(const Tuple& tuple) const
-{
-    const long fid = id(tuple, PrimitiveType::Face);
-    ConstAccessor<long> ha = get_cell_hash_accessor();
-    return ha.scalar_attribute(fid) == tuple.m_hash;
+    const long cid = tuple.m_global_cid;
+    return tuple.m_hash == get_cell_hash(cid, hash_accessor);
```

Comment:
to be honest Mesh could have a `Mesh::is_hash_valid(const Tuple&) -> bool` because all implementations just read off the global cid

Reply:
Done. I feel like we really need a _Mesh_impl.hpp_ soon because we have plenty of tiny functions and if we do not use link time optimization that might create quite an overhead.